draft-ietf-quic-http.md

```diff
@@ -1395,6 +1395,10 @@ settings to have any meaning upon receipt.
 Because the setting has no defined meaning, the value of the setting can be any
 value the implementation selects.
 
+Setting identifiers which were used in HTTP/2 where there is no corresponding
+HTTP/3 setting have also been reserved ({{iana-settings}}). These settings MUST
+NOT be sent, and receipt MAY be treated as an error of type H3_SETTINGS_ERROR.
```

Comment:
The `MAY` here could lead to confusion down the road. Is there any reason for us to not be prescriptive here? I think it would be safer to clearly have either `MUST treat as error` or `MUST ignore` here.

Reply:
We've used MUST NOT send / MAY error elsewhere when we haven't wanted to mandate checking.  It's enough of a deterrent to the bad behavior, when the requirement to treat as an error isn't vital to interop.  That said, it also isn't difficult to do the check, so if there's consensus to require it, I'm not opposed to the suggestion.

Reply:
I would be in favor of `MUST treat as error`. This isn't hard to detect, and it seems quite reasonable to close the connection on such errors.